examples/simple-genai-server/main.go

```diff
@@ -88,6 +90,20 @@ func main() {
 		}
 		simNpc = &snpc
 	}
+	if fid := os.Getenv("FromID"); fid != "" {
```

Comment:
nit: It looks like this pattern is prevalent here already but it's really strange to see environment variables that are not in ALL_UPPER_CASE_WITH_UNDERSCORES. I think we can fix this in a follow-on if you want to leave a TODO, though.

Reply:
QQ: should the actual env variables also be ALL_UPPER_CASE_? I.e. should it be `PORT := flag.String("port"...` or `PORT := flag.String("PORT"...`

Reply:
* The flag name *should* be lower-case-with-dashes: https://github.com/googleforgames/agones/blob/main/cmd/controller/main.go#L58-L83
  * _That said_, you're following a nearer convention: https://github.com/googleforgames/agones/blob/main/examples/simple-game-server/main.go#L41-L50 .. so honestly just keep the flag names. Let's fight that battle another day.
* The Go variable name should follow Go style, which is mixedCase: https://go.dev/doc/effective_go#mixed-caps
* The environment variable should be in UPPER_CASE_WITH_UNDERSCORES: https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names